index.js

```diff
@@ -24,8 +25,9 @@ module.exports = function standardVersion (argv) {
     }
   }
 
+  let args = Object.assign({}, defaults, argv)
   let pkg
-  bump.pkgFiles.forEach((filename) => {
+  args.packageFiles.forEach((filename) => {
```

Comment:
@jbottigliero If the list of package files is configurable, _updaters_ should be allowed as for bump files. Otherwise, this configuration will not work:

```json
"packageFiles": [
    "pom.xml"
],
"bumpFiles": [
    {
        "filename": "pom.xml",
        "updater": "xml-updater.js"
    }
]
```

Reply:
Updated - thanks!